ElasticExporter logs flush failures as error

**Describe the bug**

On periodically flush the elastic exporter is logged as error 

https://github.com/camunda-cloud/zeebe/blob/develop/exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporter.java#L114-L115

This cause several error reports on our reporting tool. 

We changed the log level for exceptions thrown by exporters here https://github.com/camunda-cloud/zeebe/issues/5564, but we didn't touched the elastic exporter.



**To Reproduce**

Make elastic full

**Expected behavior**
This is normally a transitive thing, which should disappear. Warning level would be enough here.


**Log/Stacktrace**
If possible add the full stacktrace or Zeebe log which contains the issue. 

<details><summary>Full Stacktrace</summary>
 <p>

```
io.zeebe.exporter.ElasticsearchExporterException: Failed to flush bulk
at io.zeebe.exporter.ElasticsearchClient.flush (ElasticsearchClient.java:152)
at io.zeebe.exporter.ElasticsearchExporter.flush (ElasticsearchExporter.java:125)
at io.zeebe.exporter.ElasticsearchExporter.flushAndReschedule (ElasticsearchExporter.java:112)
at io.zeebe.util.sched.ActorJob.invoke (ActorJob.java:76)
at io.zeebe.util.sched.ActorJob.execute (ActorJob.java:39)
at io.zeebe.util.sched.ActorTask.execute (ActorTask.java:122)
at io.zeebe.util.sched.ActorThread.executeCurrentTask (ActorThread.java:94)
at io.zeebe.util.sched.ActorThread.doWork (ActorThread.java:78)
at io.zeebe.util.sched.ActorThread.run (ActorThread.java:191)
Caused by: java.net.ConnectException: Timeout connecting to [elasticsearch-master/10.4.2.247:9200]
at org.elasticsearch.client.RestClient$SyncResponseListener.get (RestClient.java:959)
at org.elasticsearch.client.RestClient.performRequest (RestClient.java:233)
at io.zeebe.exporter.ElasticsearchClient.exportBulk (ElasticsearchClient.java:188)
at io.zeebe.exporter.ElasticsearchClient.flush (ElasticsearchClient.java:149)
Caused by: java.net.ConnectException: Timeout connecting to [elasticsearch-master/10.4.2.247:9200]
at org.apache.http.nio.pool.RouteSpecificPool.timeout (RouteSpecificPool.java:169)
at org.apache.http.nio.pool.AbstractNIOConnPool.requestTimeout (AbstractNIOConnPool.java:632)
at org.apache.http.nio.pool.AbstractNIOConnPool$InternalSessionRequestCallback.timeout (AbstractNIOConnPool.java:898)
at org.apache.http.impl.nio.reactor.SessionRequestImpl.timeout (SessionRequestImpl.java:198)
at org.apache.http.impl.nio.reactor.DefaultConnectingIOReactor.processTimeouts (DefaultConnectingIOReactor.java:213)
at org.apache.http.impl.nio.reactor.DefaultConnectingIOReactor.processEvents (DefaultConnectingIOReactor.java:158)
at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor.execute (AbstractMultiworkerIOReactor.java:351)
at org.apache.http.impl.nio.conn.PoolingNHttpClientConnectionManager.execute (PoolingNHttpClientConnectionManager.java:221)
at org.apache.http.impl.nio.client.CloseableHttpAsyncClientBase$1.run (CloseableHttpAsyncClientBase.java:64)
at java.lang.Thread.run (Thread.java:834)
```

</p>
</details>


**Environment:**
 - OS: benchmark
 - Zeebe Version: SNAPSHOT
 - Configuration: benchmark

--- a/exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporter.java
+++ b/exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporter.java
@@ -111,8 +111,7 @@ public class ElasticsearchExporter implements Exporter {
     try {
       flush();
     } catch (final Exception e) {
-      log.error(
-          "Unexpected exception occurred on periodically flushing bulk, will retry later.", e);
+      log.warn("Unexpected exception occurred on periodically flushing bulk, will retry later.", e);
     }
     scheduleDelayedFlush();
   }

